### DISCUSSION TITLE
Episode 196: 🇬🇧 Algorithms in APL with Aaron Hsu

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#95] [Episode]
Episode Link: https://adspthepodcast.com/2024/08/23/Episode-196.html

Feel free to write comments or ask questions 🙂

**Comment by kentf** [comment]
This is a comment for episode 198, but there is no discussion for that episode yet 🤷‍♂️.

1. I think I'd want to try the quiz, but I'd hate to have my voice on the podcast 😅.

2. Feature (podcast) request; ADSP B-sides - Bryce Unfiltered, where 10x Bryce can talk as much as he wants about Henry the clock, coffee table, and elevator pigeons.
